lib/std/fs/wasi.zig

```diff
@@ -22,6 +23,30 @@ pub const Preopens = struct {
         }
         return null;
     }
+
+    pub fn findDir(p: Preopens, full_path: []const u8, flags: std.fs.Dir.OpenDirOptions) std.fs.Dir.OpenError!std.fs.Dir {
```

Comment:
Can you please add some doc comments for this? I wrote the above function and I apologize for my lack of doc comments. But now I'm looking at this later and at a glance I don't understand what this function is for.

Reply:
heh, fair enough, I guess we were all too eager to merge 🙏 apologies

Reply:
is it ok if I add docs to `find()` too?